impeller/entity/entity_pass.cc

```diff
@@ -340,7 +343,7 @@ bool EntityPass::Render(ContentContext& renderer,
   // there's no need to set up a stencil attachment on the root render target.
   if (reads_from_onscreen_backdrop) {
     auto offscreen_target = CreateRenderTarget(
-        renderer, root_render_target.GetRenderTargetSize(),
+        renderer, root_render_target.GetRenderTargetSize(), /*mip_count=*/4,
```

Comment:
There's a bit of guidance around generating the optimal `mip_count` in https://github.com/flutter/flutter/issues/140949. But one baby step on the way there would be to just generate the full mipmap whenever we know a backdrop filter is going to read the pass texture:
1. Add a param to `CreateRenderTarget` for the mip count and snake it through
2. For the `CreateRenderTarget` call in `EntityPass::Render()`, set the mip count to `root_render_target.GetRenderTargetSize().MipCount()`
3. For the `CreateRenderTarget` call in `EntityPass::GetEntityForElement`, set the mip count to `(subpass->GetTotalPassReads(renderer) > 0) ? subpass_size.MipCount() : 1u`

At that point, you could even hide these params behind `IMPELLER_ENABLE_NEW_GAUSSIAN_FILTER` and land this. Then later we can optimize by working out the optimal mip count if necessary.